modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,204 @@
+----
+apiVersion: operator.openshift.io/v1
+kind: ClusterCSIDriver
+metadata:
+ name: csi.vsphere.vmware.com
+spec:
+ logLevel: Normal
+ managementState: Managed
+ observedConfig: null
+ operatorLogLevel: Normal
+ unsupportedConfigOverrides: null
+ driverConfig: 
+  driverType: vSphere <1>
+    vSphere:
+      topologyCategories: <2>
+      - openshift-zone
+      - openshift-region
+----
+<1> Ensure that `driverType` is set to `vSphere`.
+<2> `openshift-zone` and `openshift-region` categories created earlier in vCenter.
+
+. Verify that `CSINode` object has topology keys:
++
+[source, terminal]
+----
+~ $ oc get csinode
+----
++
+Example output:
++
+[source, terminal]
+----
+NAME                        DRIVERS     AGE
+co8-4s88d-infra-2m5vd       1           27m
+co8-4s88d-master-0          1           70m
+co8-4s88d-master-1          1           70m
+co8-4s88d-master-2          1           70m
+co8-4s88d-worker-j2hmg      1           47m
+co8-4s88d-worker-mbb46      1           47m
+co8-4s88d-worker-zlk7d      1           47m
+----
++
+[source, terminal]
+----
+~ $ oc get csinode co8-4s88d-worker-j2hmg -o yaml
+----
++
+Exmaple output:
++
+[source, terminal]
+----
+...
+spec:
+    drivers:
+    - allocatable:
+        count: 59
+    name: csi-vsphere.vmware.com
+    nodeID: co8-4s88d-worker-j2hmg
+    topologyKeys: <1>
+    - topology.csi.vmware.com/openshift-zone
+    - topology.csi.vmware.com/openshift-region
+----
+<1> Topology keys from vSphere `openshift-zone` and `openshift-region` catagories.
++
+[NOTE]
+=====
+`CSINode` objects might take some time to receive updated topology information. After the driver is updated, `CSINode` objects should have topology keys in them.
+=====
+
+. Create a tag to assign to datastores across failure domains:
++
+When an {product-title} spans more than one failure domain, the datastore might not be shared across those failure domains, which is where topology-aware provisioning of persistent volumes (PVs) is useful. 
++
+.. In vCenter, create a category called `openshift-zonal-datastore-cat`. This example uses the name "openshift-zonal-datastore-cat" for this category; however, you can use any name you like.
+.. In vCenter, create a tag called `openshift-zonal-datastore` for the `openshift-zonal-datastore-cat` category.
+.. Assign the `openshift-zonal-datastore` tag to each datastore in a failure domain that would be considered for dynamic provisioning.
++
+====
+NOTE: You can use any names you like for categories and tags. The names used in this example are provided as recommendations. Ensure that the tags and categories that you define uniquely identify only datastores that are shared with all hosts in the {product-title} cluster.
+====
+
+. Create a storage policy that targets the tag-based datastores in each failure domain:
+.. In vCenter, from the main menu, click *Policies and Profiles*.
+.. On the *Policies and Profiles* page, in the left navigation pane, click *VM Storage Policies*.
```

Comment:
In an attempt to adhere to https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences, I think we should remove "left" and just say "navigation pane." Depending on the screen, it could be up, down, right, etc. Also some people might use screen readers which further complicates things. Just a suggestion